src/app/shared/shared.module.ts

```diff
@@ -21,6 +22,7 @@ import { TimerComponent } from './components/timer/timer.component';
     ButtonComponent,
     ButtonsStylingDirective,
     TimerComponent,
+    NavigationComponent,
   ],
   imports: [CommonModule, FormsModule],
```

Comment:
As i understand you can imports your module right away. Because it's shared module and this module will give his components to others.

Reply:
Yes, you made a good point. As I noticed, you connected that component in. Thank you